Formula/pgpool-ii.rb

```diff
@@ -32,13 +33,35 @@ class PgpoolIi < Formula
   end
 
   def install
-    system "./configure", "--disable-dependency-tracking", "--prefix=#{prefix}",
-                          "--sysconfdir=#{etc}"
+    system "./configure", *std_configure_args,
+                          "--sysconfdir=#{etc}",
+                          "--with-memcached=#{Formula["libmemcached"].opt_include}"
     system "make", "install"
+
+    # Install conf file with low enough memory limits for default `memqcache_method = 'shmem'`
+    etc.install etc/"pgpool.conf.sample" => "pgpool.conf"
+    inreplace etc/"pgpool.conf" do |s|
```

Comment:
Let's `inreplace` the sample first before installing it to avoid tampering with the user's `pgpool.conf`.

Reply:
good idea, done